torch/csrc/distributed/autograd/context/dist_autograd_context.cpp

```diff
@@ -94,10 +96,23 @@ void DistAutogradContext::setGraphTask(
 
 void DistAutogradContext::addOutstandingRpc(
     const std::shared_ptr<rpc::FutureMessage>& futureMessage) {
+  futureMessage->addCallback(std::bind(
```

Comment:
If possible, we should prefer using lambdas instead of `std::bind`

Reply:
https://stackoverflow.com/questions/17363003/why-use-stdbind-over-lambdas-in-c14